src/commands/community/nft/stats.ts

```diff
@@ -0,0 +1,41 @@
+import { Command } from "types/common"
+import community from "adapters/community"
+import { PREFIX } from "utils/constants"
+import { composeEmbedMessage } from "utils/discordEmbed"
+import { getEmoji } from "utils/common"
+
+const command: Command = {
+  id: "nft_stats",
+  command: "stats",
+  brief: "show total collections added",
+  category: "Community",
+  run: async function (msg) {
+    const res = await community.getCollectionCount()
+    return {
+      messageOptions: {
+        embeds: [
+          composeEmbedMessage(msg, {
+            title: "NFT Collections statistic",
```

Comment:
format like this
![image](https://user-images.githubusercontent.com/39881166/181477679-62e1384d-9123-4cec-9108-7e10af758005.png)